### PR TITLE
Minor: HTML5 > HTML

### PR DESCRIPTION
**Pull request by estelle**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ The output of this code is as follows:
 
 > **Note:** You can also see this running live as [forms-test.html](https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/html-css/forms-test.html) on GitHub (see the [source code](https://github.com/mdn/learning-area/blob/main/tools-testing/cross-browser-testing/html-css/forms-test.html) also).
 
-If you view the example on a supporting browser like desktop/Android Chrome or iOS Safari, you'll see the special widgets/features in action as you try to input data. On a non-supporting platform such Internet Explorer, the inputs will just fallback to normal text inputs, so at least the user can still enter some information.
+If you view the example on a supporting browser, you'll see the UI features in action as you try to input data. On devices with dynamic keyboards, type specific keypads will be displayed. On a non-supporting browser lik Internet Explorer, the inputs will just fallback to normal text inputs, meaning the user can still enter the correct information.
```
Comment (estelle) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md:161]:
```suggestion
If you view the example on a supporting browser, you'll see the UI features in action as you try to input data. On devices with dynamic keyboards, type-specific keypads will be displayed. On a non-supporting browser like Internet Explorer, the inputs will just default to normal text inputs, meaning the user can still enter the correct information.
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md [hunk 2]
```diff
@@ -293,15 +293,15 @@ There are many other issues you'll encounter with HTML and CSS, making knowledge
...
 - Other supporting reference material, e.g. the [Guide to media types and formats on the web](/en-US/docs/Web/Media/Formats),
 - Useful tutorials that solve specific problems, for example, [Creating a cross-browser video player](/en-US/docs/Web/Guide/Audio_and_video_delivery/cross_browser_video_player).
 
 [caniuse.com](https://caniuse.com/) provides support information, along with a few useful external resource links. For example, see <https://caniuse.com/#search=video> (you just have to enter the feature you are searching for into the text box).
 
-[stackoverflow.com](https://stackoverflow.com/) (SO) is a forum site where you can ask questions and have fellow developers share their solutions, look up previous posts, and help other developers. You are advised to look and see if there is an answer to your question already, before posting a new question. For example, we searched for "cross browser html5 video" on SO, and very quickly came up with [HTML5 Video with full cross browser compatibility](https://stackoverflow.com/questions/16212510/html5-video-with-full-cross-browser-compatibility).
```
Comment (estelle) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md:304]: this article was over 9 years old, so found something newer.

File: files/en-us/learn/forms/test_your_skills_colon__html5_controls/index.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ This aim of this skill test is to assess whether you've understood our [The HTML
 >
 > If you get stuck, then ask us for help — see the [Assessment or further help](#assessment_or_further_help) section at the bottom of this page.
 
-## HTML5 controls 1
+## HTML controls 1
 
-First let's explore some of the new HTML5 input types. Create appropriate inputs for a user to update their details for:
+First let's explore some input types. Create appropriate inputs for a user to update their details for:
```
Comment (estelle) [inline files/en-us/learn/forms/test_your_skills_colon__html5_controls/index.md:22]:
```suggestion
First, let's explore some input types. Create appropriate inputs for a user to update their details for:
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md [hunk 4]
```diff
@@ -135,7 +135,7 @@ For example:
...
 
-Newer form elements also exhibit fallback qualities — HTML5 introduced some special [`<input>`](/en-US/docs/Web/HTML/Element/input) types for inputting specific information into forms, such as times, dates, colors, numbers, etc. These are very useful, particularly on mobile platforms, where providing a pain-free way of entering data is very important for the user experience. Supporting platforms provide special UI widgets when these input types are used, such as a calendar widget for entering dates.
+Another example is form elements. When new [`<input>`](/en-US/docs/Web/HTML/Element/input) types were introduced for inputting specific information into forms, such as times, dates, colors, numbers, etc, if a browser didn't support the new feature, the browser used the default of `type="text"`. Input types were added, which are very useful, particularly on mobile platforms, where providing a pain-free way of entering data is very important for the user experience. Platforms provide different UI widgets depending on the input type used, such as a calendar widget for entering dates. Should a browser not support an input type, the user can still enter the required data.
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md:138]:
```suggestion
Another example is form elements. When new [`<input>`](/en-US/docs/Web/HTML/Element/input) types were introduced for inputting specific information into forms, such as times, dates, colors, numbers, etc, if a browser didn't support the new feature, the browser used the default of `type="text"`. Input types were added, which are very useful, particularly on mobile platforms, where providing a pain-free way of entering data is very important for the user experience. Platforms provide different UI widgets depending on the input type, such as a calendar widget for entering dates. Should a browser not support an input type, the user can still enter the required data.
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**